### DISCUSSION TITLE
Log in to an account for which the email doesn't exist anymore

### DISCUSSION THREAD

**Discussion by lonnort** [community/community#31834] [Other Feature Feedback, Questions, & Ideas] [answered]
Recently I came across an old github account of mine. This account was made with an email-address that was provided to me by my high school. Now that I no longer study there, the email-address no longer exists and therefore I can't log in to that github account because the verification code ends up in a black hole. Is there any way around this, or will that account be doomed to remain floating out there, inactive, forever?

**Comment by melvincwng** [comment, marked as answer]
@lonnort Hi Lennard 👋🏻 , for account specific issues, please kindly GitHub Support over [here](https://support.github.com/contact/account) for help.

Due to the sensitive nature of account related matters, the best suggestion I have for you is to contact and try to resolve with GitHub Support directly.

Hope this answers your question 😄

**Reply by lonnort** [reply]
Thank you, that might help (guess it depends on github support 😅).

I completely overlooked it on the support page.

**Reply by themegabyte** [reply]
Hey @lonnort In a similar situation, did you get this resolved? Let me know! 

Cheers.

**Reply by lonnort** [reply]
@themegabyte Yes, I was able to get it resolved. I did by following the provided link, and sending GitHub Support a request. I hope they will be able to help you out as well.

**Comment by neofactor2** [comment]
How do create a ticket if you cant login?

**Comment by VE3ZRD** [comment]
I do believe github support is dead!!!  I put in a ticket a month and a half ago with no response as yet.
anyone that has had a response in 2024, count yourself lucky!!
Ticket #2619691outstanding since the middle of February.
As i have no collaborators I am now  way behind in publishing my work,

**Reply by ghost** [reply]
Same with me
